extension/src/paymentHandler/manual-capture.handler.js

```diff
@@ -2,26 +2,22 @@ const pU = require('./payment-utils')
 const { manualCapture } = require('../service/web-component-service')
 const {
   CTP_INTERACTION_TYPE_MANUAL_CAPTURE,
-  CTP_CUSTOM_FIELD_MANUAL_CAPTURE_RESPONSE
 } = require('../config/constants')
 
 async function execute (paymentObject) {
-  const manualCaptureRequestObj = JSON.parse(paymentObject.custom.fields.manualCaptureRequest)
-  const { request, response } = await manualCapture(manualCaptureRequestObj)
+  const chargeInitialTransaction = pU.getChargeTransactionInitial(paymentObject)
+  const authorizationSuccessTransaction = pU.getAuthorizationTransactionSuccess(paymentObject)
+  const { request, response } = await manualCapture(chargeInitialTransaction,
+    authorizationSuccessTransaction.interactionId)
   const actions = [
     pU.createAddInterfaceInteractionAction({
       request, response, type: CTP_INTERACTION_TYPE_MANUAL_CAPTURE
     }),
-    pU.createSetCustomFieldAction(CTP_CUSTOM_FIELD_MANUAL_CAPTURE_RESPONSE, response)
   ]
-  if (!response.errorCode && response.pspReference)
-    actions.push(pU.createAddTransactionAction({
-      type: 'Charge',
-      state: 'Pending',
-      amount: manualCaptureRequestObj.modificationAmount.value,
-      currency: manualCaptureRequestObj.modificationAmount.currency,
-      interactionId: response.pspReference
-    }))
+  if (!response.errorCode && response.pspReference) {
+    actions.push(pU.createChangeTransactionStateAction(chargeInitialTransaction.id, 'Pending'))
+    actions.push(pU.createChangeTransactionInteractionId(chargeInitialTransaction.id, response.pspReference))
+  }
 
```

Comment:
can you remove those extra empty lines, in general, can you check the format?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/341/commits/87dd9e28d8b0b70a5ee1bc232f52f2b12cb4ce9f